sanic/app.py

```diff
@@ -457,6 +457,49 @@ def add_route(
         )(handler)
         return handler
 
+    def make_websocket_handler(self,handler,subprotocols=None):
```

Comment:
I want to think on this name a little. We might be able to find something more approachable and understandable.

Reply:
what about websockify ?